ext/curl/lib/hostip6.c

```diff
@@ -86,6 +91,7 @@ bool Curl_ipv6works(struct connectdata *conn)
     }
     return (ipv6_works>0)?TRUE:FALSE;
   }
+  return TRUE;
```

Comment:
Why was this required? As far as I can see, the execution can never reach here but not sure if all indentations look correct in github's PR comparison view.

Reply:
not needed. removed. thanks for noticing.